src/components/forms/groups/Create.js

```diff
@@ -3,11 +3,61 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { reduxForm, Field, stopSubmit } from 'redux-form';
 
-import Input from 'components/Input';
+import styles from './styles/index.scss';
 import Button from 'components/Button';
 
 const CREATE_GROUP_FORM = 'createGroup';
 
+const renderInputField = ({ input, label, placeholder, meta: { touched, error } }) => (
+  <div>
+    <div>
+      <label>{label}</label>
+    </div>
+    <input className="form-control" {...input} placeholder={placeholder} />
+    {touched && <div className={`text-danger ${styles.error}`}>{error}</div>}
+  </div>
```

Comment:
Why you don't use `components/Input`?

You can change what you need if it doesn't feed your needs